src/rule/RuleInputs/DateRule.vue

```diff
@@ -94,49 +84,49 @@ export default Vue.extend({
         dateFormat: 'Y-m-d H:i',
       },
       weekdays: [
-        {value: 1, en: 'Monday', fr: 'Lundi'},
-        {value: 2, en: 'Tuesday', fr: 'Mardi'},
-        {value: 3, en: 'Wednesday', fr: 'Mercredi'},
-        {value: 4, en: 'Thursday', fr: 'Jeudi'},
-        {value: 5, en: 'Friday', fr: 'Vendredi'},
-        {value: 6, en: 'Saturday', fr: 'Samedi'},
-        {value: 0, en: 'Sunday', fr: 'Dimanche'},
+        { value: 1, en: 'Monday', fr: 'Lundi' },
+        { value: 2, en: 'Tuesday', fr: 'Mardi' },
+        { value: 3, en: 'Wednesday', fr: 'Mercredi' },
+        { value: 4, en: 'Thursday', fr: 'Jeudi' },
+        { value: 5, en: 'Friday', fr: 'Vendredi' },
+        { value: 6, en: 'Saturday', fr: 'Samedi' },
+        { value: 0, en: 'Sunday', fr: 'Dimanche' },
       ],
       operator: EngineSubOperators.EqualTo,
       operatorList: DateOperatorOptions,
-    }
+    };
   },
   created() {
     // @ts-expect-error DataGrid is  set by plugin config
-    this.lang = (this.$DataGrid.lang as RegistrationLanguage) ?? RegistrationLanguage.FR
+    this.lang = (this.$DataGrid.lang as RegistrationLanguage) ?? RegistrationLanguage.FR;
```

Comment:
- [ ] `:lang=` should probably come from the datagrid/plugin configuration,
- [ ] `RegistrationLanguage` should be removed entirely

Reply:
FYI, you can : 

![image](https://user-images.githubusercontent.com/794449/226514534-da2da557-ed39-4289-b2bb-aefc843c2e43.png)


if you want to check that later